.travis.yml

```diff
@@ -3,4 +3,4 @@ sudo: false
 dist: trusty
 
 script:
-  - npm run lint .
+  - npm run lint . && npm test
```

Comment:
Ah, I thought it would stop in the first non zero return. I'll change it later today. Thanks